client/src/components/DetailCard.js

```diff
@@ -0,0 +1,90 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import GlobalStyles from './GlobalStyles';
+import Bin from '../assets/bin.svg';
+import Pen from '../assets/pen.svg';
+import MethodButton from './MethodButton';
+
+const Card = styled.div`
+  width: 343px;
+  height: 570px;
+  border-radius: 10px 10px 10px 10px;
+  border: 3px solid #b9b9b9;
+  color: white;
+  display: grid;
+  grid-template-rows: 10% 8% 8% 8% auto 10%;
+  grid-template-columns: 30% auto 10% 10%;
+  align-items: center;
+`;
+
+const Heading = styled.h1`
+  text-align: left;
+  font-size: 1.25rem;
+  font-weight: 400;
+  color: #011e62;
```

Comment:
It would be good to save your colors in a file or theme, makes work easier :)

Reply:
okay thanks :)